### ISSUE TITLE
Create new folder (from quickMove) should display it at top of recent list

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#163]
When we create a new subfolder via the quickMove ⇒ New folder, I would expect this to be displayed on the "Recent Folders" menu, preferably at the top of results.

Use case:
I was moving an email from the "calm" mailing list into Subscriptions/Calm, which didn't exist at the time. Once entering in the quickMove field Sub/Calm I used the menu entry "create subfolder ... subscriptions » Calm".
![image](https://user-images.githubusercontent.com/6314674/120922145-675af700-c6bf-11eb-8b76-620a7e224e14.png)

this generated the new folder and moved the mail to it. I then wanted to jump to the new folder via "Recent" expecting the new folder to be on top of the result list:

![image](https://user-images.githubusercontent.com/6314674/120922199-9ffad080-c6bf-11eb-9fd6-ed31770bb438.png)

however the new folder was not listed in "Recent". 

Expected behavior: show the newly generated folder on top of this list. 
IIRC I do internally update the folder timestamp using Util.touch() but it doesn't show up.

**Comment by RealRaven2000** [comment]
Debugging this - the function that generates the folder on completion calls quickMove.execute() which in turn calls `touch()`:

```
  touch: function touch(folder) {
    const util = QuickFolders.Util,
          FLAGS = util.FolderFlags;
    try {
      ...
      if (folder.SetMRUTime)
        folder.SetMRUTime();
      else {
        let ct = parseInt(new Date().getTime() / 1000); // current time in secs
        folder.setStringProperty("MRUTime", ct);
      }
```

In Tb78, the function SetMRUTime doesn't exist but the stgring property "MRUTime" is being set to the current time stamp. So the question is why is it not shown in recent folders?

**Comment by RealRaven2000** [comment]
Now testing in Tb 68 - during generation of the Recent folders menu from the navigation bar, (after creating the new folder) the call to `QuickFolders.Util.generateMRUlist()`, the function to return the timestamp `util.getMruTime(folder)` returns an empty string - the string proprety "MRUTime" is missing / empty.

**Comment by RealRaven2000** [comment]
I tried various remedies, but I think my code correctly sets the timestamp after folder generation. However I believe even though moving the file works after having created the folder, Thunderbird does something to reset the timestamp - this can still be code that "tidies up" from the folder generation event - as a workaround I am delaying the call to "touch()" which sets the timestamp by 800ms - this appears to have the desired effect on my own profile (over 1400 folders). It's not ideal but a workaround for now.

**Comment by fredstluka** [comment]
Yeah, delaying the touch() by 800ms sounds ugly, but sometimes you have no choice.  Go for it!

--Fred

**Comment by RealRaven2000** [comment]
Fixed in 5.6.1
